### ISSUE TITLE
Office.js can't even import its own generated OOXML

### ISSUE THREAD

**Issue by Shujee** [OfficeDev/office-js#2304]
I haven't been able to make Office.js import my OOXML content. I was trying to follow instructions from the [official documentation](https://docs.microsoft.com/en-us/office/dev/add-ins/word/create-better-add-ins-for-word-with-office-open-xml). My OOXML injects an image in the primary header of the document. Whenever I run the following line:

`context.document.body.insertOoxml(myOOXML, Word.InsertLocation.replace);`

I'm always met with the cryptic error message:

> Uncaught (in promise) RichApi.Error: GeneralException
>    at new RuntimeError (word-win32-16.01.debug.js:10747)
>    at RequestContext.ClientRequestContext.processRequestExecutorResponseMessage (word-win32-16.01.debug.js:13995)
>    at word-win32-16.01.debug.js:13900

For a long time, I kept looking for semantic problems in my OOXML and whether I was including correct `pkg:part`s in it. Nothing worked.

Later I tried to created a Word document manually (inserting an image in the header) and then used Office.js to export its OOXML, using:

`var ooxml = context.document.body.getOoxml();`

This correctly generated document's OOXML, which I then tried to import into a new document, using `insertOoxml` call shown above. To my surprise, even that import fails and gives the same cryptic and unhelpful **GeneralException** message. So it doesn't seem to be related to the correctness of OOXML being imported.

## Your Environment
* Platform: PC desktop
* Host: Word
* Office version number: Office 2019 standard desktop (2110 Build 16.0.14527.20234, 32-bit)
* Operating System: Win10 (10.0.19042.1348)
 
## Expected behavior
`insertOoxml` should work correctly and import the supplied OOXML into the doc.

## Current behavior
Nothing is imported and instead we see **RichApi.Error: GeneralException** in the console. It doesn't even work for the OOXML that was generated by itself.

## Steps to reproduce
1. Create a JS add-in project in VS2019.
2. Write `import` and `export` functions in Javascript portion calling `context.document.body.getOoxml();` and `context.document.body.insertOoxml(ooxml.value, Word.InsertLocation.replace);` therein. `export` should save the returned OOXML in a variable and `import` should read that value and inject it into document.
2. Add a couple of buttons on your HTML page (that will be shown in the task pane) to call import and export functions.
3. Run the add-in. 
4. In the default document, add a pictuere into the primary header. 
5. We'll use this image to cover the entire page (as a background), so set wrap mode to "Behind Text" and position to 0,0 with respect to **page**. Set width and height to 8.5,11 and lock aspect ratio to false. You'll need to do this twice because of a bug in Size and Position dialog.
5. Once done, close Header and now press your Export button and then the Import button.
6. There you see the dreaded **GeneralException** message.

## Context
My add-in needs to take an image file from the user and then set it as the background of the document. I'm trying to insert the image into the header and then cover entire page with it (wrap: behind text, position: 0,0, size: page width/height).

**Comment by yihuax-msft** [comment]
@Shujee, What is the real source code you used? What is the error message? You put a picture in the header, but you are still reading and writing the main document. So your picture is not used at all.

**Comment by Shujee** [comment]
@yihuax Hi, thanks for the input. Answering your questions:

1. Full error message is already there in my original post above.
2. I have shared the minimal steps to reproduce the problem. Basic issue is that `insertOoxml` throws this **GeneralException** even for its own xml, i.e. if you call `getOoxml` function, store its output somewhere and then call `insertOoxml` and supply the store xml to it, you'll get this exception.
3. I have managed to solve my issue though. It looks like one needs to supply `<w:document>` with `<w:pStyle w:val="Header"/>` to insert something in the header. This is counter-intuitive because if I call `getOoxml` on my header, the output creates an `<w:hdr>` package.
